models/repo_generate.go

```diff
@@ -210,3 +330,36 @@ func GenerateIssueLabels(ctx DBContext, templateRepo, generateRepo *Repository)
 	}
 	return nil
 }
+
+func generateExpansion(src string, templateRepo, generateRepo *Repository) string {
+	return os.Expand(src, func(key string) string {
+		switch key {
+		case "REPO_NAME":
+			return generateRepo.Name
+		case "TEMPLATE_NAME":
+			return templateRepo.Name
+		case "REPO_DESCRIPTION":
+			return generateRepo.Description
+		case "TEMPLATE_DESCRIPTION":
+			return templateRepo.Description
+		case "REPO_OWNER":
+			return generateRepo.MustOwnerName()
+		case "TEMPLATE_OWNER":
+			return templateRepo.MustOwnerName()
+		case "REPO_LINK":
+			return generateRepo.Link()
+		case "TEMPLATE_LINK":
+			return templateRepo.Link()
+		case "REPO_HTTPS_URL":
+			return generateRepo.CloneLink().HTTPS
+		case "TEMPLATE_HTTPS_URL":
+			return templateRepo.CloneLink().HTTPS
+		case "REPO_SSH_URL":
+			return generateRepo.CloneLink().SSH
+		case "TEMPLATE_SSH_URL":
+			return templateRepo.CloneLink().SSH
+		default:
+			return key
```

Comment:
```suggestion
			return "${" + key + "}"
```

IMHO keys that are not supported/recognized should not modify the text at all. That's because `${token}` is valid syntax for several languages (e.g. PHP).

Reply:
I almost think this should actually return something that stands out more so that it gets noticed and fixed in the theoretical upstream template.  
Based on my comment regarding escaping expansions, anything that accidentally gets expanded should stand out so it can be either fixed or escaped imo

Reply:
Just as a follow-up here, if variables are escaped, returning `key` will leave them "unaltered"  
e.g. `$${VAR}` will be `${VAR}` and `$$VAR` will be `$VAR`